doc/README.md

```diff
@@ -100,10 +100,13 @@ The MongooseIM platform documentation:
     * [Release/Installation configuration](user-guide/release_config.md)
     * [High-level Architecture](user-guide/MongooseIM-High-level-Architecture.md) from single to multiple node setup to multi-datacenter
     * [How to build](user-guide/How-to-build.md) from source code
+    * [Setting up Push Notifications](user-guide/Push-notifications.md)
+    * [How to: Set up Fennec (TURN/STUN server) and see how it works (a.k.a. ICE demo)](user-guide/ICE_tutorial.md)
```

Comment:
On the title and the filename:
> * [How to: Set up Fennec (TURN/STUN server) and see how it works (a.k.a. ICE demo)](user-guide/ICE_tutorial.md)

Currently (latest, at the time fo this writing) on https://mongooseim.readthedocs.io/en/latest/ we have in the side menu:
> * How to set up Push Notifications with MongoosePush
> * How to set up MongooseICE (ICE/TURN/STUN server)

I suggest:
* How to set up MongoosePush for push notifications
* How to set up MongooseICE  for ICE/TURN/STUN
for consistency, readability, and to respect the "principle of least astonishment" (POLA).

I also suggest to rename:
* `user-guide/ICE_tutorial.md` to `user-guide/ICE-STUN-TURN_tutorial.md` 
* `user-guide/Push-notifications.md`to `user-guide/Push-notifications_tutorial.md`
or something similar for semantic URL.

Reply:
I think renaming these files is out of scope of this PR, as it would be good to change ICE_tutorial folder name but it involves updating links...

Reply:
I REALLY prefer the "How to:" approach
If you want to drop the  ":" you have to keep everything else short and simple
This "How to set up MongooseICE for ICE/TURN/STUN
for consistency, readability, and to respect the "principle of least astonishment" (POLA)."  sounds weird
So in my opinion:
[link]How to: Set up MongooseICE[/link] for ICE/TURN/STUN
[link]How to: Set up MongoosePush[/link] for Push Notifications
Or:
[link]How to set up MongooseICE[/link] for ICE/TURN/STUN
[link]How to set up MongoosePush[/link] for Push Notifications
But nothing more